.gitignore

```diff
@@ -22,3 +22,6 @@ layer-sdk/src
 # ignore SAM CLI created files/dirs
 .aws-sam/
 samconfig.toml
+
+#template file 
+template.yml
```

Comment:
Why this? We do need to track the main template file :)

Reply:
Did you mean to include this in the `cdk/typescript/.gitignore` file?

Reply:
> Did you mean to include this in the `cdk/typescript/.gitignore` file?

Yes, I will add this file there.